ui/src/app/pages/login/account-creation/account-creation-survey.tsx

```diff
@@ -180,42 +182,43 @@ or another sexual and/or gender minority?'>
       </Section>
       {/*Sex at birth section*/}
       <Section header='Sex at birth'>
-        <FlexRow style={{flexWrap: 'wrap'}}>
+        <FlexColumn style={{justifyContent: 'flex-start', flexWrap: 'wrap', height: '5rem', width: '26rem'}}>
```

Comment:
Can we extract the style out since its being used at L148 as well?

Reply:
Just noticed value of height from is different from L148, maybe take height as a parameter or override it here?